src/olympia/scanners/migrations/0046_delete_waffle_switch.py

```diff
@@ -0,0 +1,14 @@
+from django.db import migrations
+
+
+def remove_waffle_switch(apps, schema_editor):
+    Switch = apps.get_model('waffle', 'Switch')
+
+    Switch.objects.filter(name='enable-wat').delete()
```

Comment:
We can't do that until after the code has been removed, otherwise during the push we would potentially enable wat for a very short moment during which the old code is still running on the migrated db.

So this migration (and maybe the other one, it matters less) should be in a follow-up issue scheduled for the following push after the one in which this PR lands.

Reply:
(Sorry, should have caught this earlier when you mentioned running a clean-up migration)

Reply:
> We can't do that until after the code has been removed, otherwise during the push we would potentially enable wat for a very short moment during which the old code is still running on the migrated db.

How could that happen? I am not sure the pref has been created on -prod for instance.

Reply:
Err you're right, disregard me. The default would be for the switch not to be active, so we're fine...